receiver/mongodbreceiver/metrics.go

```diff
@@ -314,6 +334,127 @@ func (s *mongodbScraper) recordNetworkCount(now pcommon.Timestamp, doc bson.M, e
 	}
 }
 
+// Lock Metrics are only supported by MongoDB v3.2+
+func (s *mongodbScraper) recordLockAcquireCounts(now pcommon.Timestamp, doc bson.M, dBName string, errs *scrapererror.ScrapeErrors) {
+	mongo30, _ := version.NewVersion("3.2")
+	if s.mongoVersion.LessThan(mongo30) {
```

Comment:
nit: Let's have the var name correlate to the version. Same issue is present on the other new `record` funcs.
```suggestion
	mongo32, _ := version.NewVersion("3.2")
	if s.mongoVersion.LessThan(mongo32) {
```